utils/merge_and_push_translations.sh

```diff
@@ -122,21 +130,24 @@ undo_oneliner_diffs
 
 if [ -z "$(git status -s)" ]; then
   echo "Run completed, nothing to commit."
-  exit 0
+else
+  # Stage changes
+  # - Translations
+  git add po/*/*.po po/*/*.pot data/i18n/locales/*.json xdg/translations/*.json || true
+  # - Authors
+  git add data/txts/*.lua || true
+  # - Locale data
+  git add data/i18n/*.lua || true
+  # - Appdata
+  git add xdg/org.widelands.Widelands.appdata.xml xdg/org.widelands.Widelands.desktop || true
+  # - Statistics
+  git add data/i18n/translation_stats.conf || true
+
+  # Commit
+  git commit -m "Updated translations catalogs."
```

Comment:
```suggestion
  git commit -m "Updated translations catalogs and statistics."
```

or so…

Maybe appdata and authors should be done for first commit. I didn't check, but don't think they're affected by `.pot` changes.

Reply:
The changed pot files will be uploaded to transifex and then the po files downloaded again. So I am totally unsure about the effects.

Reply:
We have the appdata and the list of translators in JSON format, which are not affected by gettext. (I'm not sure how the appdata source can get updated.) `tx` is configured to pull these along with .po files, and `update_appdata.py` and `update_authors.py` merges them. That's why they should be run with the first .po pull, not with the .pot push and consequent pull.

Reply:
Now `update_appdata.py` and `update_authors.py` run on the 1st commit (translations). And on the 2nd (catalogs) if a pot file has changed (or no po file had changed).